### DISCUSSION TITLE
[帮帮]名字和头像怎么换

### DISCUSSION THREAD

**Discussion by Feixiao0423** [share121/inter-knot#2471] [其他]
刚注册进来，想改个名字和头像结果怎么找都找不到方法换，有人指点一下吗

**Comment by cpu644** [comment]
你好！这里是绳网技术支援团队。请进入github，左键你的头像（右上角），找到your profile / 你的概要，然后在里面修改你的头像（不可大于1mb）和用户名。

**Comment by cpu644** [comment]
如果不行那就是我忘了。反正这的名字/头像和github名字头像是同步的 自己百度看看。

**Comment by Qweulin** [comment]
也许你需要[这个](https://github.com/maboloshi/github-chinese/raw/gh-pages/main.user.js)汉化github的油猴脚本

**Reply by Feixiao0423** [reply]
名字在个人质料那改了，头像地下也显示了，但他还是显示数字

**Comment by Feixiao0423** [comment]
谢谢已经摸索出来了
